manifests/package/install.pp

```diff
@@ -7,21 +7,17 @@
 #
 # * Justin Lambert <mailto:jlambert@letsevenup.com>
 #
-class artifactory::install {
+class artifactory::package::install inherits artifactory::params {
 
   if $caller_module_name != $module_name {
     fail("Use of private class ${name} by ${caller_module_name}")
   }
 
-  File {
-    require => Package['artifactory'],
```

Comment:
This needs to be here (or require put on the link resource) for "random" ordering resource ordering to succeed since there's a symlink created in the artifactory package home dir.